package.json

```diff
@@ -60,7 +60,7 @@
     "@inrupt/internal-test-env": "^3.1.0",
     "@inrupt/jest-jsdom-polyfills": "^3.0.2",
     "@inrupt/solid-client-authn-node": "^2.0.0",
-    "@playwright/test": "^1.40.1",
+    "@playwright/test": ">= 1.37.0, < 1.38.0",
```

Comment:
I'm not sure about the syntax of that version range expression: https://semver.npmjs.com/ complains about it (although it does what we expect it to in practice). I think `~1.37.0` is semantically equivalent, and I've seen versions pinned to the minor with this syntax more often.